src/main/java/duke/exception/IllegalDescriptionException.java

```diff
@@ -0,0 +1,16 @@
+package duke.exception;
+
+/**
+ * IllegalDescriptionException is an exception relating to invalid description input.
+ *
+ * @author totsukatomofumi
+ */
+public class IllegalDescriptionException extends IllegalArgumentException {
+    /**
+     * Constructs an IllegalDescriptionException with the specified detail message.
+     * @param s the detail message.
+     */
+    public IllegalDescriptionException(String s) {
+        super(s);
+    }
+}
```

Comment:
You may want to consider making a single DukeException for all your exceptions, since all of your exceptions are essentially the same

Throwing of the DukeException could be done inside the if-else statement (with different messages for different errors)